lib/ts/recipe/passwordless/recipe.ts

```diff
@@ -237,11 +239,13 @@ export default class Recipe extends RecipeModule {
                 ? {
                       email: input.email,
                       userInputCode,
+                      tenantId: input.tenantId,
                       userContext: input.userContext,
                   }
                 : {
                       phoneNumber: input.phoneNumber,
                       userInputCode,
+                      tenantId: input.tenantId,
                       userContext: input.userContext,
                   }
         );
```

Comment:
need to add tenantId in the generated magic link below

Reply:
Done